elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/SourceSettingsBase.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) since 2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.elasticsearch
+
+import akka.util.JavaDurationConverters._
+import java.util.concurrent.TimeUnit
+
+import akka.stream.alpakka.elasticsearch.ElasticsearchConnectionSettings
+
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * Configure Elastiscsearch/Opensearch sources.
```

Comment:
```suggestion
 * Configure Elasticsearch/OpenSearch sources.
```

I'm going to be a bit irritating about this, but there's two irritating conventions to deal with!  When using these project names with cased letters, the correct names are:

* `Elasticsearch` without any internal capital S
* `OpenSearch` **with** an internal capital S

There's no major consequence to getting it wrong, except that we can get stuck with inconsistently named classes (seeing `ElasticSearch` and  `Opensearch`).  In my opinion, it's worth sticking to each project's naming convention.

Reply:
:+1: done, thanks @RyanSkraba !